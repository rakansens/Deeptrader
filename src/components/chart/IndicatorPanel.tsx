"use client";

import { forwardRef } from "react";
import type { HTMLAttributes } from "react";
import { cn } from "@/lib/utils";

export interface IndicatorPanelProps extends HTMLAttributes<HTMLDivElement> {}

const IndicatorPanel = forwardRef<HTMLDivElement, IndicatorPanelProps>(
  ({ className, ...props }, ref) => {
    return (
      <div
        ref={ref}
        className={cn("w-full bg-background border-t border-border", className)}
        {...props}
      />
    );
  },
);
IndicatorPanel.displayName = "IndicatorPanel";

export default IndicatorPanel;
