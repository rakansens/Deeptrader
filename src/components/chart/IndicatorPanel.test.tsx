import { render } from "@testing-library/react";
import IndicatorPanel from "./IndicatorPanel";

describe("IndicatorPanel", () => {
  it("matches snapshot", () => {
    const { container } = render(<IndicatorPanel />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
