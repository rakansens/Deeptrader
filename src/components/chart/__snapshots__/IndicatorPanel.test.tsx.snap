// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndicatorPanel matches snapshot 1`] = `
<div
  class="w-full bg-background border-t border-border"
/>
`;
